libs/loggers/web_hook_logger.py

```diff
@@ -29,3 +36,58 @@ def log_objects(self, objects, violating_objects, violating_objects_index_list,
             requests.put(self.web_hook_endpoint, data=request_data)
         except ConnectionError:
             logger.error(f"Connection with endpoint {self.web_hook_endpoint} can't be established")
+
+    def update(self, cv_image, objects, post_processing_data, fps):
+        violating_objects = post_processing_data.get("violating_objects", [])
+        if not self.web_hook_endpoint:
+            return
+        if time.time() - self.submited_time > self.time_interval:
+            # Get timeline which is used for as Timestamp
+            now = datetime.now()
+            current_time = now.strftime("%Y-%m-%d %H:%M:%S")
+            # Process objects
+            objects_formated = self.format_objects(objects)
```

Comment:
This function is very similar to the csv logger. Could we abstract more of it to a superclass? I think the format_objects function is the same

Reply:
Yes, you are right. I created a `RawDataLogger`  parent of `FileSystemLogger` and `WebHookLogger`